config/filter.d/zzz-generic-example.conf

```diff
@@ -0,0 +1,17 @@
+# Fail2Ban generic example resp. test filter 
```

Comment:
so it would be shipped with fail2ban installations... but if solely for testing, why not to keep it in testing?